modules/client-configuration/pages/clients-rh.adoc

```diff
@@ -155,161 +135,131 @@ subscription-manager repos
 You can use these repository URLs to create custom repositories.
 This allows you to mirror only the content you need to manage your clients.
 
-[IMPORTANT]
-====
-For {redhat} 8 clients, add both the ``BaseOS`` and ``Appstream`` channels.
-You will require packages from both channels.
-If you do not add both channels, you will not be able to create the bootstrap repository, due to missing packages.
-====
-
 [IMPORTANT]
 ====
 You can only create custom versions of {redhat} repositories if you have the correct entitlements in your {redhat} Portal.
 ====
 
 
+The details you need for this procedure are:
 
-.Procedure: Creating Custom Repositories
+[[redhat-repos-manual]]
+[cols="1,1", options="header"]
+.Red Hat Custom Repository Settings
+|===
+| Option                 | Parameter
+| Repository URL         | The content URL provided by {redhat} CDN
+| Has Signed Metadata?   | Uncheck all {redhat} Enterprise repositories
+| SSL CA Certificate     | [systemitem]``redhat-uep``
+| SSL Client Certificate | [systemitem]``Entitlement-Cert-date``
+| SSL Client Key         | ``Entitlement-Key-date``
+|===
 
-. On the {productname} Server {webui}, navigate to menu:Software[Manage > Repositories].
-. Click btn:[Create Repository] and set these parameters for the entitlement certificate:
-* In the [guimenu]``Repository Label`` field, type [systemitem]``rhel-7-server-rpms``.
-* In the [guimenu]``Repository URL`` field, type the URL of the repository to mirror.
-For example, [systemitem]``https://cdn.redhat.com/content/dist/rhel/server/7/7Server/x86_64/os/``.
-* In the [guimenu]``Has Signed Metadata?`` field, uncheck all Red Hat Enterprise Repositories.
-* In the [guimenu]``SSL CA Certificate`` field, select [systemitem]``redhat-uep``.
-* In the [guimenu]``SSL Client Certificate`` field, select [systemitem]``Entitlement-Cert-date``.
-* In the [guimenu]``SSL Client Key`` field, select [systemitem]``Entitlement-Key-date``.
-* Leave all other fields as the default values.
-. Click btn:[Create Repository].
-. Repeat for every repository you want to define.
+include::snippets/manual_repos.adoc[]
 
 
-When you have created the custom repositories, you can create corresponding custom channels.
 
+The channels you need for this procedure are:
 
+[[redhat-channels-custom]]
+[cols="1,1,1", options="header"]
+.Red Hat Custom Channels
+|===
+| OS Version | Base Channel      | Updates Channel
+| {redhat} 6 | RHEL6 Base x86_64 | -
+| {redhat} 7 | RHEL7 Base x86_64 | -
+| {redhat} 8 | RHEL8 Base x86_64 | rhel8-appstream for x86_64
```

Comment:
It does not seem to exist anymore.

Reply:
Do you mean the appstream channel? What is it now?

Reply:
See the screenshot in my bug https://bugzilla.suse.com/attachment.cgi?id=838669. I don't see "RHEL8 Base x86_64" anymore. You can reproduce by opening the setup wizard and 

Please test this feature as @juliogonzalez asked in the bug (https://bugzilla.suse.com/show_bug.cgi?id=1172803#c1). We mean it.

Now we have one unique channel for both SLES ES 8 and RHEL 8. What changes between SLES ES 8 and RHEL 8 is the child channels the user checks.  This needs to be documented.